### PR TITLE
Show 'no info' box when left-click didn't have results

### PR DESCRIPTION
**Pull request by gjn**
When there are queryable layers on the map, we display a hand when the users hovers over a feature. A feature is detected by openlayers when a layer pixel is not empty (client side detection). We then issue an identify request. The client side detection and the identify request don't fully corrrespond and we don't have results. The application does nothing, which is confusing to the user because he saw the hand (cursor style).

This PR show a `No further information` box in such cases which auto-closes in 3 seconds. This is the same exact behaviour as in map.wanderland.ch. Also, the translations were taken from there.

Fixes https://github.com/geoadmin/mf-geoadmin3/issues/2823

[Testlink](https://mf-geoadmin3.dev.bgdi.ch/gjn_noinfo/?lang=de&topic=ech&bgLayer=ch.swisstopo.swissimage&layers=ch.swisstopo.zeitreihen,ch.bfs.gebaeude_wohnungs_register,ch.bafu.wrz-wildruhezonen_portal,ch.swisstopo.swisstlm3d-wanderwege,ch.swisstopo.fixpunkte-agnes,ch.astra.wanderland&layers_visibility=false,false,false,false,true,true&layers_timestamp=18641231,,,,,&X=177319.45&Y=633928.77&zoom=8)

Note: translated files look like huge update because it's the first time we use new translation script.

### CODE REVIEW

**Comment by gjn** [comment]
@davidoesch Can you test and give the go?

@oterral Can you review?

**Comment by davidoesch** [comment]
+1 perfect

File: src/components/popup/PopupService.js [hunk 1]
```diff
@@ -45,8 +45,17 @@ goog.require('ga_draggable_directive');
...
-      Popup.prototype.open = function() {
+      Popup.prototype.open = function(autoclose) {
+        var that = this;
         this.scope.toggle = true;
+        if (autoclose) {
+          $timeout(function() {
+            if (that && that.scope && that.scope.toggle) {
+              that.close();
+            }
+          }, autoclose);
```
Comment (oterral) [inline src/components/popup/PopupService.js]: Can we change the `autoclose` parameter to `autocloseDelay` . Without I had the feeling this varianble was a boolean.

**Comment by oterral** [comment]
lgtm I've changed the variable name.

**Comment by oterral** [comment]
I tested a bit and it seems the 'No information' popup is always displayed: 

[Test with no info box](https://mf-geoadmin3.dev.bgdi.ch/gjn_noinfo/?lang=de&topic=ech&bgLayer=ch.swisstopo.swissimage&layers=ch.swisstopo.zeitreihen,ch.bfs.gebaeude_wohnungs_register,ch.bafu.wrz-wildruhezonen_portal,ch.swisstopo.swisstlm3d-wanderwege,ch.swisstopo.fixpunkte-agnes,ch.astra.wanderland&layers_visibility=false,false,false,false,true,true&layers_timestamp=18641231,,,,,&X=170324.50&Y=631625.20&zoom=13)  ---> click on the line is the middle -> no info box shown


[Test without no info box](https://mf-geoadmin3.dev.bgdi.ch/?lang=de&topic=ech&bgLayer=ch.swisstopo.swissimage&layers=ch.swisstopo.zeitreihen,ch.bfs.gebaeude_wohnungs_register,ch.bafu.wrz-wildruhezonen_portal,ch.swisstopo.swisstlm3d-wanderwege,ch.swisstopo.fixpunkte-agnes,ch.astra.wanderland&layers_visibility=false,false,false,false,true,true&layers_timestamp=18641231,,,,,&X=170324.50&Y=631625.20&zoom=13)  ---> click on the line is the middle -> an info box with results is shown

**Comment by oterral** [comment]
I've added a  commit to open the 'no info' popup when all identify request are done.

**Comment by gjn** [comment]
I had different functionality in mind. No Info box is shown only when clicked-with-no-results when cursor indicated that there are result (hand is shown). When a click is down with standard arrow, then this is not shown. This would avoid to show this message everytime a user clicks the map.

Maybe too complicated?

**Comment by oterral** [comment]
No I have  another PR for this but I waited this one was merged

**Comment by davidoesch** [comment]
Agree with gjn

**Pull request merged**